kgtk/visualize/visualize_api.py

```diff
@@ -178,15 +198,19 @@ def compute_visualization_graph(self):
 
             count = 0
             color_set = {}
+            literal_count = 0
+            literal_list = []
+
 
             for row in kr:
+                temp_edge = {}
                 if self.node_file is None:
-                    if '@' in row[l1]:
+                    if '@' in row[l1] and self.auto_remove_wikidata_language_tag:
                         nodes.add((row[n1], row[l1][1:row[l1].find('@') - 1]))
```

Comment:
Use https://github.com/usc-isi-i2/kgtk/blob/master/kgtk/kgtkformat.py#L129 to do the same